ui/src/album/AlbumActions.js

```diff
@@ -28,10 +37,28 @@ const AlbumActions = ({
   const dispatch = useDispatch()
   const translate = useTranslate()
   const isDesktop = useMediaQuery((theme) => theme.breakpoints.up('md'))
+  const [playing, setPlaying] = useState(false)
```

Comment:
Why `useState`? Don't we have this information in the Redux store?

Reply:
Yes 😅 My bad, will remove it